process/process.dsc.resource.json

```diff
@@ -0,0 +1,27 @@
+{
+    "manifestVersion": "1.0",
+    "description": "Returns information about running processes.",
+    "type": "Microsoft/ProcessList",
+    "version": "0.1.0",
+    "get": {
+        "executable": "process",
+        "args": [
+          "get"
+        ]
+    },
+    "set": {
```

Comment:
Does `set` work?  If not, don't have it declared.

Reply:
I could not come up with anything possible to set and implementation-wise simple enough that you can set on the actual running process.
But I need this `set` for testing. Currently it always returns successful status.